pkg/nsx/services/common/store.go

```diff
@@ -152,3 +152,42 @@ func (service *Service) InitializeResourceStore(wg *sync.WaitGroup, fatalErrors
 		}
 	}
 }
+
+// InitializeProjectResourceStore is the method to query all the various resources from nsx-t side and
+// save them to the store, we could use it to cache all the resources when process starts.
+func (service *Service) InitializeProjectResourceStore(wg *sync.WaitGroup, fatalErrors chan error, resourceTypeValue string, store Store) {
```

Comment:
It seems that the existing func `InitializeResourceStore` also does the same thing. What's the difference?

Reply:
I also refactored it at https://github.com/vmware-tanzu/nsx-operator/pull/172/files